provider/src/main/java/com/synopsys/integration/alert/provider/blackduck/collector/builder/LicenseLimitMessageBuilder.java

```diff
@@ -56,8 +56,8 @@ public LicenseLimitMessageBuilder() {
 
             messageContentBuilder
                 .applyCommonData(commonMessageData)
-                .applyTopic(MessageBuilderConstants.LABEL_LICENSE_LIMIT_MESSAGE, notificationContent.getMessage())
-                .applySubTopic(MessageBuilderConstants.LABEL_USAGE_INFO, usageMessage)
+                .applyProject(MessageBuilderConstants.LABEL_LICENSE_LIMIT_MESSAGE, notificationContent.getMessage())
```

Comment:
I thought license limit notification did not apply to projects and project versions?

Reply:
Yeah, LicenseLimit was and continues to be a Hack within Alert. Not sure what else to do with the rename though.

Reply:
We don't necessarily need to solve it for this PR but that is something we should tackle this version.